timeline-chart/src/layer/time-graph-markers-chart.ts

```diff
@@ -0,0 +1,122 @@
+import { TimeGraphMarkerState } from "../components/time-graph-marker";
+import { TimeGraphRow } from "../components/time-graph-row";
+import { TimelineChart } from "../time-graph-model";
+import { TimeGraphRowController } from "../time-graph-row-controller";
+import { TimeGraphChart, TimeGraphChartProviders } from "./time-graph-chart";
+
+/**
+ * Extension of the time-graph-chart, modified to hold logic for the markers layer.
+ * @param id string
+ * @param providers TimeGraphChartProviders
+ * @param rowController TimeGraphRowController
+ */
+export class TimeGraphMarkersChart extends TimeGraphChart {
+
+    private _selectedMarker?: TimelineChart.TimeGraphMarkerState;
+
+    constructor(id: string, protected providers: TimeGraphChartProviders, protected rowController: TimeGraphRowController) {
+        super(id, providers, rowController);
+        
+        this.registerMouseInteractions({
+            click: el => {
+                if (el instanceof TimeGraphMarkerState) {
```

Comment:
I'm not able to select any marker state. I set a breakpoint but it never hit. Not sure what the reason is.

Reply:
I'm not sure; this is working for me.  Maybe try again when I push my changes.

Reply:
I was trying to select a symbol marker. I realized that this feature is for the marker row. I'm able to select a marker now.